test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Utilities.cs

```diff
@@ -24,14 +25,15 @@ internal static class Utilities
         private static int NextPort = BasePort;
         private static object PortLock = new object();
         internal static readonly TimeSpan DefaultTimeout = TimeSpan.FromSeconds(15);
+        internal static readonly int WriteRetryLimit = 1000;
 
         internal static IServer CreateHttpServer(out string baseAddress, RequestDelegate app)
         {
             string root;
             return CreateDynamicHttpServer(string.Empty, out root, out baseAddress, options => { }, app);
         }
 
-        internal static IServer CreateHttpServer(out string baseAddress, Action<HttpSysOptions> configureOptions, RequestDelegate app)
+        internal static IServer CreateHttpServer(out string baseAddress, RequestDelegate app, Action<HttpSysOptions> configureOptions)
```

Comment:
Whats the point of the re-order?

Reply:
Consistency with the other overload. I was hitting ambiguity issues where it was confusing the Action parameters.